snapcraft/tests/unit/commands/test_version.py

```diff
@@ -19,7 +19,11 @@
 
 
 class VersionCommandTestCase(CommandBaseTestCase):
-
     def test_has_version(self):
         result = self.run_command(['--version'])
         self.assertThat(result.exit_code, Equals(0))
+
+    def test_has_version_without_hyphens(self):
+        result = self.run_command(['version'])
+        print(result.output)
```

Comment:
In general, tests shouldn't print anything - instead I would recommend you do something like `self.assertThat(result.output, Equals(snapcraft.__version__))` which will only output the value if it fails.